libraries/chain/include/sophiatx/chain/get_config.hpp

```diff
@@ -0,0 +1,154 @@
+#pragma once
+
+#include <sophiatx/protocol/config.hpp>
+#include <sophiatx/protocol/protocol_config.hpp>
+#include <sophiatx/protocol/asset.hpp>
+#include <sophiatx/protocol/types.hpp>
+#include <sophiatx/protocol/version.hpp>
+#include <sophiatx/chain/genesis_state.hpp>
+
+namespace sophiatx { namespace chain {
+
+class sophiatx_config {
```

Comment:
if it was class siophiatx_config : private fc::mutable_variant_object {
...
}
you would not need class member config_